### DISCUSSION TITLE
Workaround - Ability to set Dashboard defaults and visibility per user (not per device or maybe in addition to)

### DISCUSSION THREAD

**Discussion by baylanger** [home-assistant/frontend#11316] [Configuration]
Related to discussion #11278 , I didn’t want to hijack that discussion to discuss possible workaround since then.

Still considering myself new here, perhaps some of you can chip in some comments in here.

My current approach for this issue until such feature has been implemented goes like this.

1) haos (VM or RPi) - this would be the main install with *all* integrations.
2) hass docker instance(s) for wife & children.  Would configure minimum integrations.  For example, I would not include Tesla, Synology nas, MQTT and others.  They’ll be able to control locks, lights, perhaps see cameras, etc and if there’s a need to restrict anything, will remove the device.

I don’t mind to have a docker instance for such purpose.

Do you think that’s a good approach?  Seems this could work in many cases.

Open to all suggestions!

**Comment by Fenrir88** [comment]
Have you tried any other approach? I made a dotnet mvc that tracks some of my sensors and has some buttons for scripts over home assistant API but UI is not my strong.
